plugins/metrics/pkg/agent/drivers/opni_manager_otel/otel_driver.go

```diff
@@ -253,7 +252,7 @@ func init() {
 	drivers.NodeDrivers.Register("opni-manager-otel", func(_ context.Context, opts ...driverutil.Option) (drivers.MetricsNodeDriver, error) {
 		options := OTELNodeDriverOptions{
 			Namespace: os.Getenv("POD_NAMESPACE"),
-			Logger:    logger.NewPluginLogger().Named("metrics").Named("otel"),
+			Logger:    logger.NewPluginLogger().WithGroup("metrics").WithGroup("otel"),
```

Comment:
Some plugin loggers are created during `init()`, before I can call the `identityClient` to get the agent ID corresponding to the log file. Is there another way to initialize or pass in these loggers?

Reply:
Added a `pluginWriter` that gets initialized after the agent id is available. Let me know what you think!